### PR TITLE
IDE-3991 maven init bundle move to LiferayMavenWorkspaceCreateListener

### PR DESCRIPTION
**Pull request by haoyisun**
IDE-3991 maven init bundle move to LiferayMavenWorkspaceCreateListener

### CODE REVIEW

File: maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenWorkspaceCreateListener.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+					IPath workspaceProjectPath = workspaceProject.getLocation();
+
+					IPath bundlesPath = workspaceProjectPath.append("bundles");
+
+					if (LiferayServerCore.isPortalBundlePath(bundlesPath)) {
+						ServerUtil.addPortalRuntimeAndServer(workspaceProjectName, bundlesPath, monitor);
+					}
+				}
+				catch (CoreException ce) {
+					LiferayMavenCore.logError("Init LiferayBundle failed", ce);
```
Comment (jtydhr88) [inline maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenWorkspaceCreateListener.java]: should not give the **try** with the whole code, we need to think where the try

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewMavenLiferayWorkspaceOpTests.java [hunk 2]
```diff
@@ -76,6 +79,18 @@ public void testNewMavenLiferayWorkspaceSetUrl() throws Exception
...
+
+            String jobName = job.getName();
+
+            if (jobName.equals("Init Liferay Bundle")) {
+                return true;
+            }
+
+            return false;
+
+        }, 30*60*1000);
```
Comment (jtydhr88) [inline maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewMavenLiferayWorkspaceOpTests.java]: sf

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewMavenLiferayWorkspaceOpTests.java [hunk 3]
```diff
@@ -115,6 +130,18 @@ public void testNewMavenLiferayWorkspaceInitBundle() throws Exception
...
```
Comment (jtydhr88) [inline maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewMavenLiferayWorkspaceOpTests.java]: sf

**Comment by jtydhr88** [comment]
see comment and resend, thanks.

**Comment by haoyisun** [comment]
ok ,get it

**Comment by jtydhr88** [comment]
sent to Greg, thanks.

**Pull request closed without merging**